src/ZprimeSelectionModule.cxx

```diff
@@ -141,7 +144,13 @@ bool ZprimeSelectionModule::process(Event & event){
   jetlepton_cleaner->process(event);
   jetER_smearer->process(event);
   jet_cleaner1->process(event); // jets w/ pt>25 GeV for lepton-2Dcut
-  bool pass_twodcut = twodcut_sel->passes(event); // lepton 2D-cut
+  //Trigger selection
+  bool ele_Trigger_selection = ele_Trigger->passes(event);
+  bool mu_Trigger_selection = mu_Trigger->passes(event);
+  if (!ele_Trigger_selection || !mu_Trigger_selection || (mu_Trigger_selection && ele_Trigger_selection)) return false;
```

Comment:
just one comment: in the run1 framework, we were passing the HLT path as a string from the xml file. that's probably best, b/c it avoids loading two trigger selections when only one is needed (because we run the selection separately for muon and electron channel)